features/fields_extensie.feature

```diff
@@ -16,48 +16,54 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
     En persoon met burgerservicenummer 999992077 is overleden
 
   Scenario: gevraagd gegeven is in onderzoek en geen fields gebruikt
-    Als persoon wordt gevraagd met "/ingeschrevenpersonen/999994086"
```

Comment:
Om deze feature te kunnen automaten, moeten de stap definities worden herschreven zoals de andere aangepaste features. De achtergrond moet dan als volgt worden herschreven

```
    Gegeven het systeem heeft een persoon met de volgende gegevens
    | naam                | waarde    |
    | burgerservicenummer | 999994086 |
    En de persoon heeft de volgende geboorte gegevens
    | naam                                      | waarde   |
    | datum (01.03.10)                          | 19860401 |
    | land.code                                 | 6030     |
    | plaats.code                               | 0518     |
    | inOnderzoek.aanduidingGegevensInOnderzoek | TODO     |
    | inOnderzoek.datum                         | 20200601 |
    En het systeem heeft een persoon met de volgende gegevens
    | naam                          | waarde    |
    | burgerservicenummer           | 999993483 |
    | geheimhoudingPersoonsgegevens | 7         |
    En het systeem heeft een persoon met de volgende gegevens
    | naam                | waarde    |
    | burgerservicenummer | 999992077 |
    En de persoon heeft de volgende overlijden gegevens
    | naam | waarde |
    | TODO | TODO   |
    En de waardetabel 'Gemeenten' heeft de volgende waarden
    | code | omschrijving  |
    | 0518 | 's-Gravenhage |
    En de waardetabel 'Landen' heeft de volgende waarden
    | code | omschrijving |
    | 6030 | Nederland    |
```
De Achtergrond stappen moeten worden aangevuld. Minimaal de TODO velden moeten met een waarde worden gevuld. De opschortingbijhouding gegevens van de overleden persoon moet ook worden gespecificeerd in een Gegeven stap.

De titel van de scenario beginnend bij regel 18 is niet meer correct. fields is nu een verplicht input veld. Ik zou het als volgt schrijven: Geboortedatum is in onderzoek en fields=geboorte wordt gebruikt. De scenario stappen moeten dan als volgt worden herschreven

```
  Scenario: Geboortedatum is in onderzoek en fields=geboorte wordt gebruikt
    Als personen wordt gezocht met de volgende parameters
    | naam                | waarde                          |
    | type                | RaadpleegMetBurgerservicenummer |
    | burgerservicenummer | 999994086                       |
    | fields              | geboorte                        |
    Dan bevat de persoon met burgerservicenummer '999994086' de volgende geboorte gegevens
    | naam                             | waarde     |
    | datum.type                       | Datum      |
    | datum.datum                      | 1986-04-01 |
    | land.code                        | 6030       |
    | land.omschrijving                | Nederland  |
    | plaats.code                      | 0518       |
    | plaats.omschrijving              | 's-Gravenhage |
    | inOnderzoek.datumIngangOnderzoek | 2020-06-01 |
    | inOnderzoek.datum                | true       |
    | inOnderzoek.land                 |            |
    | inOnderzoek.plaats               |            |

```